support-frontend/assets/pages/new-contributions-landing/components/ContributionTypeTabs.jsx

```diff
@@ -18,6 +17,7 @@ import type { Switches } from 'helpers/settings';
 import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import { type State } from '../contributionsLandingReducer';
 import { updateContributionTypeAndPaymentMethod } from '../contributionsLandingActions';
+import type { ContributionTypes, ContributionTypeSetting } from '../../../helpers/contributions';
```

Comment:
I think I'd rather not have a relative path to `helpers` in case the file moves... I know that there are other relative paths, but they're to very local contribution-specific files rather than the global helpers